plugins/woocommerce/woocommerce.php

```diff
@@ -60,3 +61,45 @@ function wc_get_container() {
 
 // Global for backwards compatibility.
 $GLOBALS['woocommerce'] = WC();
+
+// Jetpack's Rest_Authentication needs to be initialized even before plugins_loaded.
+if ( class_exists( \Automattic\Jetpack\Connection\Rest_Authentication::class ) ) {
+	\Automattic\Jetpack\Connection\Rest_Authentication::init();
+}
+
+/**
+ * Initialize the Jetpack functionalities: connection, identity crisis, etc.
+ */
+function wc_jetpack_init() {
+	$config = new Automattic\Jetpack\Config();
+	$config->ensure(
+		'connection',
+		array(
+			'slug' => 'woocommerce',
+			'name' => __( 'WooCommerce', 'woocommerce' ),
+		)
+	);
+
+	// When only WooCommerce is active, minimize the data to send back to WP.com for supporting Woo Mobile apps.
+	$config->ensure(
```

Comment:
I think these are from WooCommerce Payments? I'm not sure if they are required 🤔

Reply:
It is in WooCommerce Payments, but also in the example repos. I think we can remove it though.

Reply:
Removed in https://github.com/woocommerce/woocommerce/pull/38674/commits/bc315156fcb6f22692b98892b90b8607c92f087a